test/www/jxcore/test/thalicryptomanagertests.js

```diff
@@ -0,0 +1,138 @@
+"use strict";
+
+require('./mockmobile');
+var fs = require('fs');
+var path = require('path');
+var crypto = require('crypto');
+var test = require('tape');
+var cryptomanager = require('../thali/thalicryptomanager');
+
+
+function deleteFolder(fileLocation, file, cb) {
+  if(file) {
+    fs.unlink(file, function(err) {
+      if(err) {
+        console.error('failed to delete the pkcs12 file - err: ', err);
+      }
+      fs.rmdir(fileLocation);
```

Comment:
What happens if there is an error? Since you are using the asynchronous version the Node.js docs say that the exception will be given to the callback. But there is no callback here. So it will silently fail. What you can do is instead use rmdirSync which should throw an exception if there is a problem.